backend/papersfeed/utils/recommendations/utils.py

```diff
@@ -142,23 +189,55 @@ def __pack_recommendations(recommendations, request_user):
 
     for recommendation in recommendations:
 
-        paper = paper_utils.get_papers(Q(id=recommendation.paper.id), request_user, 1)[0]
-        action_object = {
+        paper = paper_utils.get_papers(Q(id=recommendation.paper.id), request_user, 1)[0][0]
+        action_object_paper = {
             constants.TYPE: 'paper',
             constants.CONTENT: paper,
         }
 
-        packed_recommendation = {
+        packed_paper_recommendation = {
             constants.ID: recommendation.id,
             constants.ACTOR: {
```

Comment:
I think we can just remove ‘actor’ from responses from recommendation. It will be more efficient to both backend and frontend. If we aim to merge this PR before today’s practice session, we can leave this problem.